packages/apps/composer-app/src/shared-worker.ts

```
@@ -8,6 +8,7 @@
 // See: https://github.com/Menci/vite-plugin-wasm/issues/37
 
 onconnect = async (event) => {
```

Biome (2.4.4)

[error] A global variable should not be reassigned.

(lint/suspicious/noGlobalAssign)